AlphaWallet/TokenScriptClient/Models/AssetAttributeSyntax.swift

```diff
@@ -74,6 +74,8 @@ enum AssetAttributeSyntax: String {
             return .string(address.eip55String)
         case .string:
             return value
+        case .bytes:
+            return value
```

Comment:
The return value for `.bytes` has to be converted to string first and then wrapped as: `.string(stringConvertedFromData)`